sympy/polys/rings.py

```diff
@@ -3005,3 +3122,257 @@ def sqf_list(f, all=False):
 
     def factor_list(f):
         return f.ring.dmp_factor_list(f)
+
+def cont_prim(p, x):
+    """Returns the content and primitive part of a polynomial with respect to a
+    specified variable."""
+
+    coeffs = p.coeff_split({x})
+    cont = gcd_prs(coeffs)
+    prim = p.exquo(cont)
+    return cont, prim
```

Comment:
Is there any method that can replace `exquo` to avoid `ExactQuotientFailed`

Reply:
If `gcd_prs` is correct then it should guarantee that `exquo` would succeed because it is supposed to return a divisor. Can you give an example that leads to ExactQuotientFailed?